docker-compose.yml

```diff
@@ -32,9 +32,9 @@ services:
        - SKIP_FORCE_SSL=true
        - WEB_URL=postgis://postgres:secret@postgres/school_experience
      depends_on:
```

Comment:
Is this  `docker-compose.yml` intended to be useable outside of the tests?

The reason I ask, is bringing up the app will will blow away any data the user has - obviously postgres isn't running with a volume here so the users going to need to change the file anyway to persist data.

Reply:
It could be, might that be a separate ticket? It wasn't considered to be in scope for this work.